### DISCUSSION TITLE
Validate existing link destinations

### DISCUSSION THREAD

**Discussion by WyrdByWord** [YOURLS/YOURLS#3480] [Q&A]
Apologies if I missed anything, since this seems like it would be a common need: is there a plugin or tool for validating existing destination/original links?

**Comment by ozh** [comment]
See http://awesome.yourls.org/

**Comment by WyrdByWord** [comment]
Thanks, but I had checked there before asking the question. I see a plugin that validates a URL when submitted, but I don't see anything that will check the existing destinations and note any that are 404 or otherwise unreachable. Sorry if I am just overlooking it.
